Docs/source/developers/repo_organization.rst

```diff
@@ -16,7 +16,23 @@ The main WarpX class is WarpX, implemented in ``Source/WarpX.cpp``.
 Build system
 ------------
 
-WarpX uses the AMReX build system. Each sub-folder contains a file ``Make.package`` with the names of header files and source files to include for the build.
+WarpX uses the AMReX build system (GNUMake).
+Each sub-folder contains a file ``Make.package`` with the names of source files that are added to the build.
```

Comment:
Great PR! I would just suggest to be more explicit here about the fact that header files should not appear in the `Make.package` files. I would add something like ``Make.package should contain source files only (*.cpp). Header files (*.H) should not be listed in Make.package``